--- /usr/lib/python3/dist-packages/kibot/blender_scripts/blender_export.py.ori	2023-09-01 10:43:26.886112966 +0000
+++ /usr/lib/python3/dist-packages/kibot/blender_scripts/blender_export.py	2023-09-01 10:44:51.250423846 +0000
@@ -354,7 +354,7 @@
     bpy.ops.pcb2blender.import_pcb3d(filepath=PCB3D_file, pcb_material=args.pcb_material,
                                      import_components=args.no_components,
                                      add_solder_joints=args.solder_joints,
-                                     center_pcb=args.dont_center,
+                                     center_boards=args.dont_center,
                                      merge_materials=args.dont_merge_materials,
                                      enhance_materials=args.dont_enhance_materials,
                                      cut_boards=args.dont_cut_boards,
